charts/prometheus-redis-exporter/templates/deployment.yaml

```
@@ -55,9 +55,13 @@ spec:
             - name: REDIS_ADDR
             {{- if .Values.redisAddressConfig.enabled }}
               valueFrom:
+              {{- if .Values.redisAddressConfig.isASecret }}
```

I'd suggest to remove the indefinite article.

```suggestion
              {{- if .Values.redisAddressConfig.isSecret }}
```